src/nvim/charset.c

```diff
@@ -1022,12 +1022,12 @@ int win_lbr_chartabsize(win_T *wp, char_u *line, char_u *s, colnr_T col, int *he
       && vim_isbreak(c)
       && !vim_isbreak((int)s[1])
       && wp->w_p_wrap
-      && (wp->w_grid.Columns != 0)) {
+      && (wp->w_width_inner != 0)) {
```

Comment:
@bfredl 	any reason why grid calls it "columns" but elsewhere its "width"?

Reply:
@justinmk Not really, it shouln't be `Columns` with caps either. My idea was just to reduce the use of the variable before renaming it in anyway. Perhaps we should even use `w_width_inner` for the shared redraw paths, so that these variables can be zero when the grid is not allocated.